dangerfile.js

```diff
@@ -86,12 +86,12 @@ const componentSourceFilesModified = fileChanges.some(
 );
 
 const snapshotsModified = fileChanges.some(
-  (filePath) => filePath.endsWith('.js.snap') || filePath.endsWith('.ts.snap'),
+  (filePath) => filePath.endsWith('.js.snap') || filePath.endsWith('.tsx.snap'),
```

Comment:
What is the convention for Backpack? I guess test files that take snapshot have to be `tsx` to render the component for snapshot, but just to quickly double check.

Reply:
Good question.

ESLint TypeScript rules don't play well with JSX in `.ts` files, meaning we'd always use `.tsx` to ensure linting works without bugs - even though technically either work fine.

As you say, because of this snapshots, which are based on files with JSX in them, will be `.tsx`.

Ollie/Ana - are there any non-JSX snapshots used in Backpack? Do we need 

```
(filePath) => filePath.endsWith('.js.snap') || filePath.endsWith('.ts.snap') || filePath.endsWith('.tsx.snap'),
```

Reply:
I think we would need to capture all three just in case off the top of my head to answer the question I am not sure, but for now lets just do this catch all and once we have migrated Backpack to TS they this will be re checked anyway

Reply:
Agree 👍 

https://github.com/Skyscanner/backpack/pull/2535/commits/01057cdc2cdbb30415955ad96a9679a52857ddd0